support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -115,4 +115,27 @@ export const tests: Tests = {
 		seed: 10,
 		optimizeId: 'dQCXBc3QQIW7M1Di_qSCHw',
 	},
+	singleToRecurring: {
+		variants: [
+			{
+				id: 'control',
+			},
+			{
+				id: 'variantA',
+			},
+			{
+				id: 'variantB',
+			},
+		],
+		audiences: {
+			ALL: {
+				offset: 0,
+				size: 1,
+			},
+		},
+		isActive: false,
```

Comment:
We'll need to set `isActive` to `true` to turn it on.

Reply:
Ok, turned on now for use on prod

Reply:
Completed - turned on now for use on prod